common/schema-builder/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>smallrye-graphql-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye</groupId>
+            <artifactId>smallrye-graphql-federation-api</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I think we should move all the federation annotations directly into the api module, maybe in a separate package. @phillip-kruger: what do you think?

Reply:
Yes this should move to the api module

Reply:
Copied into the seperated `server/api/src/main/java/io/smallrye/graphql/api/federation` package. 
Please let me know if you want it to be in the root package together with the core Graph QL annotations.